mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-python_version = 3.6
+python_version = 3.13
```

Comment:
I think this should be the minimum python version so probably 3.8?

Reply:
sounds good